### PR TITLE
media-sound/snapcast Bump to v0.16.0 and GLEP81

### PR DESCRIPTION
**Pull request by thecb1**
* Version bump to 0.16.0
* EAPI=7
* GLEP 81 with user/group snapserver:snapserver + user snapclient

### CODE REVIEW

**Comment by thecb1** [comment]
@Whissi pls review :-)

**Comment by Whissi** [comment]
acct-* packages will require review through dev mailing list first, so do not merge yet.

**Comment by amak79** [comment]
> acct-* packages will require review through dev mailing list first, so do not merge yet.

Have the packages been reviewed?

**Comment by juippis** [comment]
Much has changed with acct- packages in short time. I requested your UID+GIDs, 
f56dd3e3b9fd379bbd174de2f129d2c2aa6f1d48

I'll try to go through your contribution later today.

**Comment by thecb1** [comment]
Any news from the reviewers?

File: acct-user/snapclient/snapclient-0.ebuild [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="A user for the Snapcast client component"
+ACCT_USER_ID=115
+ACCT_USER_HOME=/var/lib/snapclient
```
Comment (juippis) [inline acct-user/snapclient/snapclient-0.ebuild]: Please read 
https://devmanual.gentoo.org/ebuild-writing/users-and-groups/index.html#choosing-a-home-directory
and decide whether you really need this to be set.

File: acct-user/snapclient/snapclient-0.ebuild [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ACCT_USER_HOME_OWNER=snapclient:audio
+ACCT_USER_HOME_PERMS=0770
```
Comment (juippis) [inline acct-user/snapclient/snapclient-0.ebuild]: Same here.

File: acct-user/snapserver/snapserver-0.ebuild [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="A user for the Snapcast server component"
+ACCT_USER_ID=114
+ACCT_USER_HOME=/var/lib/snapserver
```
Comment (juippis) [inline acct-user/snapserver/snapserver-0.ebuild]: And here.

File: acct-user/snapserver/snapserver-0.ebuild [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+ACCT_USER_HOME_OWNER=snapserver:snapserver
```
Comment (juippis) [inline acct-user/snapserver/snapserver-0.ebuild]: This should be the default as well.

File: acct-user/snapserver/snapserver-0.ebuild [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+ACCT_USER_HOME_PERMS=0770
```
Comment (juippis) [inline acct-user/snapserver/snapserver-0.ebuild]: Also check this after reading the devmanual page.

File: media-sound/snapcast/snapcast-0.16.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+	if use server ; then
+		newconfd "${FILESDIR}/snapserver-0.16.0.confd" "snapserver"
+
+		insinto /etc
+		doins "${S}"/server/etc/snapserver.conf
+	fi
+}
+
+pkg_postinst() {
+	if [[ -z ${REPLACING_VERSIONS} ]]; then
```
Comment (juippis) [inline media-sound/snapcast/snapcast-0.16.0.ebuild]: What if someone had -9999 installed?

File: media-sound/snapcast/snapcast-9999.ebuild [hunk 7]
```diff
@@ -68,20 +59,27 @@ src_install() {
...
-		fperms 0770 /var/lib/snapserver
+		newconfd "${FILESDIR}/snapserver-0.16.0.confd" "snapserver"
+
+		insinto /etc
+		doins "${S}"/server/etc/snapserver.conf
+	fi
+}
+
+pkg_postinst() {
+	if [[ -z ${REPLACING_VERSIONS} ]]; then
```
Comment (juippis) [inline media-sound/snapcast/snapcast-9999.ebuild]: What if someone installs 0.16.0 and then -9999?

File: acct-user/snapclient/snapclient-0.ebuild (hunk 1 above)
Comment (amak79) [reply acct-user/snapclient/snapclient-0.ebuild]: snapclient doesn't appear to use the home dir, so it's probably safe to remove. However, the upstream Debian package does create it.

File: acct-user/snapserver/snapserver-0.ebuild (hunk 3 above)
Comment (amak79) [reply acct-user/snapserver/snapserver-0.ebuild]: snapserver creates the `server.json` file in its home dir, so this needs to be set.

File: acct-user/snapserver/snapserver-0.ebuild (hunk 5 above)
Comment (amak79) [reply acct-user/snapserver/snapserver-0.ebuild]: This home dir permissions `0755` (default) or `0750` should work.

**Comment by juippis** [comment]
Closing due to inactivity.

**Pull request closed without merging**